src/de/grobox/liberario/activities/MainActivity.java

```diff
@@ -91,6 +98,7 @@
 	private AccountHeader accountHeader;
 	private Toolbar toolbar;
 
+	@TargetApi(Build.VERSION_CODES.N_MR1)
```

Comment:
I understand the annotation as a sign to the linter, that we aware of using API that is not supported by the minimumSDKVersion. An example is `ShotcutManager`. 

I will move that out because the rest of the `onCreate` should not be affected by that.